README.rst

```diff
@@ -26,3 +26,7 @@ txjuju-daily PPAs and can be added with::
   sudo add-apt-repository -y ppa:juju/stable
   sudo add-apt-repository -y ppa:landscape/txjuju-daily
   sudo apt-get update && sudo apt-get install python-jujuclient python-txjuju
+
+In addition, you'll want to have quilt and tox installed::
+
+  sudo apt-get install quilt tox
```

Comment:
Note that tox is not strictly required, because a simple:

cd python; python -m unittest discover

should work fine as long as you have the correct dependencies in your import path (e.g. with system packages).

Tox support is mainly there for Travis.

Probably we should document both workflows, as different people have different preferences (I personally found tox overhead sometimes annoying).